File: WebApp/Front/src/models/Course.ts
import Subject from "./Subject";

export default class Course {
  courseId: number;
  courseDate: string;
  courseBeginTime: string;
  courseEndTime: string;
  subjectDto: Subject;

  constructor(
    courseId: number,
    courseDate: string,
    courseBegin: string,
    courseEnd: string,
    subject: Subject
  ) {
    this.courseId = courseId;
    this.courseDate = courseDate;
    this.courseBeginTime = courseBegin;
    this.courseEndTime = courseEnd;
    this.subjectDto = subject;
  }
} 

File: face_recognition/models/Course.py
from models.Subject import Subject

class Course:
    def __init__(self, courseId: int, courseDate: str, courseBeginTime: str, courseEndTime: str, subjectDto: Subject):
        self.courseId = courseId
        self.courseDate = courseDate
        self.courseBeginTime = courseBeginTime
        self.courseEndTime = courseEndTime
        self.subjectDto = subjectDto
